cmd/ltagent/init.go

```diff
@@ -110,23 +119,43 @@ func RunInitCmdF(cmd *cobra.Command, args []string) error {
 
 	config.UserControllerConfiguration.Type = loadtest.UserControllerGenerative
 	config.UsersConfiguration.InitialActiveUsers = 0
+	config.UsersConfiguration.PercentageOfAdminUsers = adminPercent
```

Comment:
I think this setting should be in `InstanceConfiguration` since that's what the init process deals with.
Also, for our use case, we might want to drop the percentage and go for a specific number since it might get confusing otherwise (percentage of what value? Initial users, max users, some hard-coded constant).

We also might want to overwrite the config value only if `adminPercent` was actually passed through the command line.

Reply:
Makes sense, will amend